### PR TITLE
Open-source src/kochka.com.mx

### PR DESCRIPTION
**Pull request by mrtnzlml**

### CODE REVIEW

File: src/kochka.com.mx/.github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# To get started with Dependabot version updates, you'll need to specify which
```
Comment (michalsanger) [inline src/kochka.com.mx/.github/dependabot.yml:1]: IMHO all the `src/kochka.com.mx/.github/*` files make no sense here

File: src/kochka.com.mx/src/Homepage.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    <div className={styles('background')}>
+      <div className={styles('backgroundOverlay')}>
+        <HomepageNavigation />
+        <main id="main" className={styles('root')}>
+          <div className={styles('center')}>
+            <Logo />
+
+            <div>
+              {/* TODO: popups explaining what is each of the elements (photo) */}
+              <fbt desc="catch phrase">REAL CATS + DUMPLINGS + COFFEE</fbt>
```
Comment (michalsanger) [inline src/kochka.com.mx/src/Homepage.js:23]: One could say all items referring to a menu :-) How about something like:
"COFFEE + DUMPLINGS + CUTE CATS AROUND"

File: src/kochka.com.mx/src/SocialMediaIcons.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  +vertical?: boolean,
+  +color?: string,
+  +size?: number,
+|};
+
+export default function SocialMediaIcons(props: Props): React.Node {
+  return (
+    <div className={styles('icons', props.vertical && 'iconsVertical')}>
+      <div className={styles(props.vertical ? 'iconVertical' : 'icon')}>
+        <a
```
Comment (michalsanger) [inline src/kochka.com.mx/src/SocialMediaIcons.js:21]: Interesting code scanner warning...

**Review by michalsanger — APPROVED** [review]

File: src/kochka.com.mx/.github/dependabot.yml (hunk 1 above)
Comment (mrtnzlml) [reply src/kochka.com.mx/.github/dependabot.yml:1]: You are right. 👍 A relic from the private repo.

File: src/kochka.com.mx/src/Homepage.js (hunk 2 above)
Comment (mrtnzlml) [reply src/kochka.com.mx/src/Homepage.js:23]: Thanks for the input! Open-source FTW! 🙂 We are going to discuss it and decide later.

File: src/kochka.com.mx/src/SocialMediaIcons.js (hunk 3 above)
Comment (mrtnzlml) [reply src/kochka.com.mx/src/SocialMediaIcons.js:21]: Damn, I thought we have this covered in our Eslint Config. Followup issue: https://github.com/adeira/universe/issues/2022

**Pull request merged**